src/ekobox/gen_theory.py

```diff
@@ -0,0 +1,165 @@
+import io
+import math
+import pathlib
+
+import yaml
+from banana.data import sql
+
+
+def gen_theory_card(fns, pto, initial_scale, update=None, dump=False, name=None):
```

Comment:
This should exist, at least similar, in `banana`. So if we need to expose with this signature is fine to define it, but it should at least rely on the existing one (and existing default theory)

Reply:
For the same reason of op.cards I implemented this function. Now it also rely on the default card you mentioned.